lib/atomic/activities/session.ex

```diff
@@ -59,6 +60,28 @@ defmodule Atomic.Activities.Session do
     |> cast_embed(:location, with: &Location.changeset/2)
   end
 
+  defp validate_entries_number(changeset) do
+    minimum_entries = get_change(changeset, :minimum_entries)
+    maximum_entries = get_change(changeset, :maximum_entries)
+
+    if minimum_entries && maximum_entries && minimum_entries > maximum_entries do
```

Comment:
```suggestion
    if minimum_entries > maximum_entries do
```
If those values do not exist on the changeset, `get_change/2` will return `nil` (https://hexdocs.pm/ecto/Ecto.Changeset.html#get_change/3). And `nil > nil` is `false`. Also, the validation of required fields (such as `minimum_entries` and `maximum_entries`) is made before.
I think the code is prettier like so, too 👀